src/vso-task-lib/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "vso-task-lib",
-  "version": "0.2.0",
+  "version": "0.3.0",
```

Comment:
I was planning on doing that after merging these changes and publishing vsotasklib to nom